.github/workflows/build.yml

```diff
@@ -161,15 +161,11 @@ jobs:
             cxx: clang++
             brew_packages: >
               meson ninja
-              pkgconf
               ccache
               expat
               fmt
               googletest
-              pcre2
```

Comment:
What's the rationale for removing all these packages? This means MPD will build without these features, and these features will not be verified by the CI.

Reply:
The macos-latest workflow ships with all of these packages preinstalled, except for libnfs.

Reply:
I added them because I thought it doesn't hurt to mention these, even if they are preinstalled; if some future image happens to remove them, we're still safe.